[CS2113T-T09-3] SITUS

rainish2000 opened the pull request:
Smart Inventory Tracking and Updating System (SITUS) will help restaurant inventory managers to keep track of and update their stock.

Roycius commented on docs/DeveloperGuide.md:
```diff
@@ -1,38 +1,264 @@
+
+The sequence diagram for when the user inputs `alerts all` is shown below.
+
+![image](images/AlertsAllSequenceDiagram.png)
```
Potential missing deletion of object:
* `AlertExpiringSoonCommand` and `AlertLowStockCommand` when `AlertCommand` returns, 
* `AlertCommand` when `Parser` returns.

nonrnp replied:
The activation bar should end when the method returns

![46D56980-5F28-429A-8D03-92EB986DEDF0](https://user-images.githubusercontent.com/69129667/138994542-6d66f0bb-287c-4ea4-8ad5-c8ff92dbffc6.jpg)

nonrnp replied:
Also, should there be a return from `parseAlertCommand()` method before the method called from `"alerts all"` return?

![32883FE8-594F-412E-AB72-0800232C7F25](https://user-images.githubusercontent.com/69129667/138996734-bf712741-52a7-47ad-b158-f741445d0ad9.jpg)

ruyian replied:
We need a cross for the instantiated AlertCommand once the parser returns as the instantiated object is no longer used.

ruyian replied:
The activation bar should have some difference in the length to show the execution flow

Yitching commented on docs/DeveloperGuide.md:
```diff
@@ -1,38 +1,264 @@
+
+The sequence diagram for when the user inputs `alerts all` is shown below.
+
+![image](images/AlertsAllSequenceDiagram.png)
```
![image](https://user-images.githubusercontent.com/69508658/138994476-3e1cb117-dad8-432c-9f95-bf11c90cec6c.png)

Should the activation bar end when 'both lists of ingredients' is returned

Yitching replied:
![image](https://user-images.githubusercontent.com/69508658/138994431-be4629cd-4f13-4c73-b332-a0633b60cdf0.png)

Should the activation bar of the method (ParseAlertCommand()) end before the entire class returns?

Yitching commented on docs/DeveloperGuide.md:
```diff
@@ -1,38 +1,264 @@
+
+The sequence diagram for the `AlertExpiringSoonCommand.run()` is shown below. The user can also call this via `alerts expiry`
+
+![image](images/AlertExpirySequenceDiagram.png)
```
![image](https://user-images.githubusercontent.com/69508658/138994564-7bc8bc10-e424-4b77-b0c8-f7c45e4f4023.png)

Same as above, should the activation bar end before the return statement?

nonrnp replied:
There should not be an activation bar after `expiryDate` is returned from `Ingredient` and before `getIngredientInfo()` is invoked.

Also, The activation bar for `AlertExpiringSoonCommand` should not exist after it returns.

![81DBF483-F7AB-4223-90EE-4F53B0D8B131](https://user-images.githubusercontent.com/69129667/138995156-443d7433-b29d-4930-9776-9a12a94f3c6d.jpg)

ruyian replied:
The activation bar should not get cross the OPT of the flow diagram because it is not guaranteed to execute while only expireDate is guaranteed to be returned from Ingredient

Roycius commented on docs/DeveloperGuide.md:
```diff
@@ -1,38 +1,264 @@
+
+The class diagram below shows the relationships between `Parser`, `Command` and `XYZCommand`.
+
+![image](images/ParserDiagram.png)
```
Suggestion: Add `{abstract}` to `Command` in the class diagram to make it clear that it is an abstract class.

Yitching commented on docs/DeveloperGuide.md:
```diff
@@ -1,38 +1,264 @@
+For `AlertLowStockCommand`, it is less complicated, and the sequence diagram shown below. The user can also call this via `alerts stock`
+
+
+![image](images/AlertStockSequenceDiagram.png)
```
![image](https://user-images.githubusercontent.com/69508658/138994868-f6649177-e391-4298-8876-bfc9672e46ca.png)
![image](https://user-images.githubusercontent.com/69508658/138994828-86c76747-f485-4092-b412-e941d73efbe3.png)

Same as above, should the activation bar end before the return statement?

Yitching replied:
![image](https://user-images.githubusercontent.com/69508658/138994999-9324a531-d6c2-4d3e-865f-78616ab547bf.png)

Should the activation bar not touch the frame of the loop?

Roycius commented on docs/DeveloperGuide.md:
```diff
@@ -1,38 +1,264 @@
+
+### 3.1. System Architecture
+
+![System Architecture](images/SysArch%20Diagram.png)
```
Consider changing the colour choices for `Parser` and/or `Command` as the current colour choices may make it hard for readers to read the words inside the box.

Yitching commented on docs/DeveloperGuide.md:
```diff
@@ -1,38 +1,264 @@
+The _sequence diagram_ below shows how the components interact with each other given a scenario where the user 
+enters the input `add n/carrot a/1 e/2021-11-12`
+
+![image](images/InteractionSeqDiagram.png)
```
![image](https://user-images.githubusercontent.com/69508658/138995721-776b6eed-6a60-4bcd-8b86-14b5ed03674a.png)

Should the activation bar be a full bar instead?

ruyian replied:
Command's instantiated object is no longer used. An cross should be drawn at the end.

haoyusimon replied:
![image](https://user-images.githubusercontent.com/77672961/139370198-75a956f9-20c1-41b0-8438-c06755b52c50.png)

For this part, not very sure what "i" means here. Does it refer to the carrot that will be added to the list?

nonrnp commented on docs/DeveloperGuide.md:
```diff
@@ -1,38 +1,264 @@
+For `AlertLowStockCommand`, it is less complicated, and the sequence diagram shown below. The user can also call this via `alerts stock`
+
+
+![image](images/AlertStockSequenceDiagram.png)
```
Same as the previous diagram, the activation bar should not exist in `ingredientGroup` after `totalAmount` is returned and before `toString()` is called.

The activation bar for `AlertExpiringSoonCommand` is also too long.

![D81FEC42-351A-4167-988F-E5BC7C2E9C5D](https://user-images.githubusercontent.com/69129667/138996209-5a9d7dd1-7396-4c6e-aa79-6163ef6ee85f.jpg)

Roycius commented on docs/DeveloperGuide.md:
```diff
@@ -1,38 +1,264 @@
+Then calling `delete n/ carrot e/ 25/12/2021` will remove the second entry in the `carrot` category.
+The sequence diagram below illustrates the above command example
+
+![image](images/DeleteSequenceDiagram.png)
```
The `DeleteCommand` instance in the diagram seems to not be referenced to anymore after Parser returns. Maybe add a "cross" here?
![image](https://user-images.githubusercontent.com/70126260/138997210-62b6a9d1-585b-4312-aab0-d1a8f20c831f.png)

nonrnp commented on docs/DeveloperGuide.md:
```diff
@@ -1,38 +1,264 @@
+
+Below is a partial class diagram of the `IngredientList` component
+
+![image](images/IngredientListDiagram.png)
```
I think you missed some arrowheads in the diagram

![9DA5FF6E-33A6-4C05-A757-296C9B055847](https://user-images.githubusercontent.com/69129667/138993895-84193818-2804-4bd7-b461-39dbac7c3525.jpg)

Roycius reviewed (commented):
DG is well done overall other than some potential improvements pointed out. Well done!

AayushMathur7 committed:
edits to UG/DG

AayushMathur7 committed:
Merge branch 'master' of https://github.com/AY2122S1-CS2113T-T09-3/tp

# Conflicts:
#	docs/UserGuide.md

datn02 committed:
Added exception for update and delete command

datn02 committed:
Merge pull request #103 from datn02/DucAnh-UpdateDeleteCommandFix

Added exception for update and delete command

mudkip8 committed:
Fixed Ingredient Formatting

mudkip8 committed:
Merge pull request #104 from mudkip8/master

Fixed Ingredient Formatting

mudkip8 committed:
Changed formatting for date DATE

mudkip8 committed:
Merge pull request #105 from mudkip8/master

Changed formatting for date DATE

mudkip8 committed:
Updated link to latest release

mudkip8 committed:
Merge pull request #106 from mudkip8/master

Updated link in UG to latest release

mudkip8 committed:
Updated help command

mudkip8 committed:
Merge pull request #107 from mudkip8/master

Updated help command

AayushMathur7 committed:
updated DG

mudkip8 committed:
Fixed a bug for update where decimal amounts cause an exception

mudkip8 committed:
Merge pull request #113 from mudkip8/master

Fixed a bug for update where decimal amounts cause an exception

ruyian reviewed (commented):
Overall, a great work, team!

haoyusimon commented on docs/DeveloperGuide.md:
```diff
@@ -1,38 +1,264 @@
+
+The **UI** component can be found in the `UI` package. The UI reads commands from the user, sends the command to `Main` to be executed and prints an output message upon completion of the command or if an error occurred.
+
+### 3.3. Parser component
```
Instead of talking about how Parser calls the parseXYZCommand() and executes... in 3.4. Command component, will it be better to move the parseXYZCommand() part to 3.3. Parser component? Coz it is a method of the parser, and should be considered as part of the parser component I guess.

haoyusimon commented on docs/DeveloperGuide.md:
```diff
@@ -1,38 +1,264 @@
 
+### 1.1. Purpose
+This document specifies the architectural and software design decisions in the implementation of the Smart Inventory
+Tracking and Updating System (SITUS).
```
Since there is "Instructions for manual testing" section in the DG, maybe can add additional purposes ( like showing the developers how to test/improve the software)?

haoyusimon commented on docs/DeveloperGuide.md:
```diff
@@ -1,38 +1,264 @@
+`IngredientList` object will only use `loadIngredientsFromMemory()` and `writeIngredientsToMemory()` methods
+of the storage class only when there is a change in the ingredient list of the program.
+
+## 4. Implementation
```
For the implementation section, since this DG has one of the purposes as "specifying software design decisions", I guess it is good to also mention some of the alternatives designs your team had thought of, as well as their pros and cons, to explain why your team decided to use current design.

mudkip8 committed:
Fixed Bugs where thresholds can be less than or equal to 0, causing weird behaviour

mudkip8 committed:
Changed error message for invalid threshold types for set

mudkip8 committed:
Merge pull request #165 from mudkip8/v2.1Bugfixing

Fixed Bugs where thresholds can be less than or equal to 0, causing w…

mudkip8 committed:
Added Function to check for special characters

mudkip8 committed:
Added check for special characters in find

mudkip8 committed:
Added Error Message for updating ingredient amount to less than or equal to 0

mudkip8 committed:
Added error messages to catch special characters in name, negative amount and expiry dates in the past for Add

mudkip8 committed:
Added error if subtracting negative number

mudkip8 committed:
Added imports and Messages

mudkip8 committed:
Moved the error check for add

mudkip8 committed:
Merge branch 'master' of https://github.com/AY2122S1-CS2113T-T09-3/tp into v2.1Bugfixing

mudkip8 committed:
Changed the command input expiry to 1000 years later to test the correct command

mudkip8 committed:
Fixed Date Command error message

mudkip8 committed:
Merge pull request #166 from mudkip8/v2.1Bugfixing

More bugfixing on valid inputs

mudkip8 committed:
Fixed spaces detected as parameters issue for delete

mudkip8 committed:
Changed invalidNumberFormatInput to invalidParameterFormatInput as the test case is invalid parameter type

mudkip8 committed:
Changed code for parseExpiry, parseAlert and parseSet to follow that of parseDelete

mudkip8 committed:
Merge pull request #167 from mudkip8/v2.1Bugfixing

Fixed spaces detected as parameters issue for delete

mudkip8 committed:
Merge branch 'master' of https://github.com/AY2122S1-CS2113T-T09-3/tp into v2.1Bugfixing

mudkip8 committed:
Removed duplicates for Find

mudkip8 committed:
Change date formatting to follow that of delete

mudkip8 committed:
Changed add formatting to follow delete

mudkip8 committed:
Changed method call inputs

mudkip8 committed:
Dealt with issue where repeating the word 'add' will not give an error

mudkip8 committed:
Changed subtract and update to follow style of other functions

mudkip8 committed:
Refixed all of the parser class methods

mudkip8 committed:
Added author tags

mudkip8 committed:
Merge pull request #168 from mudkip8/v2.1Bugfixing

Formatting, Input Exception catching and Adding of Author Tags

mudkip8 committed:
Set max expiry and stock thresholds

mudkip8 committed:
Created test cases for alerts, set and expire commands

mudkip8 committed:
Change error assert value

mudkip8 committed:
Merge pull request #169 from mudkip8/v2.1Bugfixing

Set max thresholds and added tests for alerts, expire and set

mudkip8 committed:
Edits based on TA comments

mudkip8 committed:
Merge pull request #170 from mudkip8/v2.1Bugfixing

Edits based on TA comments

mudkip8 committed:
Added PPP Draft

mudkip8 committed:
Merge pull request #171 from mudkip8/v2.1Bugfixing

Added PPP Draft

datn02 committed:
Fixes #117

datn02 committed:
Fix update not accepting floating point number

datn02 committed:
Fix test code format to comply with new changes

mudkip8 committed:
Changed check for alphanumeric only inputs to regex

mudkip8 committed:
Merge pull request #172 from datn02/DucAnh-Issue117

Fix issue 117

mudkip8 committed:
Merge branch 'master' of https://github.com/AY2122S1-CS2113T-T09-3/tp into v2.1Bugfixing

mudkip8 committed:
Merge pull request #173 from mudkip8/v2.1Bugfixing

Changed check for alphanumeric only inputs to regex

Commit:
edit code based on TA feedback

Commit:
add error message for incorrect index inputs

datn02 committed:
Fix update command splitting

datn02 committed:
Added exception handling

datn02 committed:
Fixes #122

datn02 committed:
Fix coding style

datn02 committed:
Merge pull request #174 from datn02/DucAnh-Issue122

Fix issue 122

Commit:
all amounts stored to 3 decimal places, fix subtraction bugs

Commit:
minor ui bug fixes

Commit:
fix error when checking for proper index format

Commit:
fix mainClassName field

Commit:
fix merge conflicts

Commit:
fix merge conflicts

Commit:
update junit test for find

rainish2000 committed:
Merge pull request #175 from nishantrai-nus/v2.1-bugfixing

V2.1 bugfixes

Commit:
fix subtraction bugs past 1 decimal place

Commit:
fix bug in last commit

rainish2000 committed:
Merge pull request #176 from nishantrai-nus/v2.1-bugfixing

fixes #149

mudkip8 committed:
Removed AlertCommandClass

mudkip8 committed:
Changed section for startup alerts to handle the 2 alert types separately

mudkip8 committed:
Added a method to print without top divider

mudkip8 committed:
Changed to handle threshold data file attacks

mudkip8 committed:
Changed parseAlerts to remove AlertsCommand

mudkip8 committed:
Added error catching for failed read/writes of threshold data

mudkip8 committed:
Fixed checkstyle Error

mudkip8 committed:
Merge pull request #177 from mudkip8/v2.1Bugfixing

Fixed threshold data read/write bugs

datn02 committed:
Add some defensive method to prevent crashing when read with wrong ingredient format

datn02 committed:
Fixes #163

datn02 committed:
Fixes #136, #164

datn02 committed:
Fixes #146
Add defensive method to ignore lines in memory file with incorrect format

datn02 committed:
Merge pull request #178 from datn02/DucAnh-v2.1StorageMinorFix

V2.1 Storage and UG fix

AayushMathur7 committed:
Merge branch 'master' of https://github.com/AY2122S1-CS2113T-T09-3/tp

# Conflicts:
#	src/main/java/seedu/situs/ingredients/IngredientList.java

datn02 committed:
Add limit to add command

mudkip8 committed:
Some edits to diagram based on DG peer review

mudkip8 committed:
Additions to DG TOC

mudkip8 committed:
Merge pull request #179 from mudkip8/DeveloperGuide

Developer guide

datn02 committed:
Fix a bug where it will display as 0 kg but not removing itself

datn02 committed:
Merge branch 'master' into DucAnh-AddLimitAmountFix

datn02 committed:
Add amount check for update command to be uniform with add command

datn02 committed:
Fix coding style
Add max ingredient threshold inside the update parser also

datn02 committed:
Merge pull request #180 from datn02/DucAnh-AddLimitAmountFix

Add limit amount and fix 0.0kg bug

ngoivanessa committed:
update colour scheme of dg

ngoivanessa committed:
Merge branch 'master' into v2.0

# Conflicts:
#	docs/DeveloperGuide.md
#	docs/diagrams/style.puml
#	docs/images/SysArch Diagram.png

ngoivanessa committed:
created add sequence diagram

ngoivanessa committed:
edit add sequence diagram

datn02 committed:
Added PPP draft

datn02 committed:
Add printing message to indicate which line is invalid in ingredients.txt

datn02 committed:
Fix coding style

datn02 committed:
Merge pull request #181 from datn02/DucAnh-PPPDraft

Add PPP draft for @datn02

AayushMathur7 committed:
Merge branch 'master' of https://github.com/AY2122S1-CS2113T-T09-3/tp

# Conflicts:
#	docs/DeveloperGuide.md
#	src/main/java/seedu/situs/ingredients/IngredientList.java

mudkip8 committed:
Added sorting by expiry date (earliest first) for expiring ingredients display

mudkip8 committed:
Merge pull request #182 from mudkip8/ExpireGrouping

Added sorting by expiry date (earliest first) for expiring ingredient…

mudkip8 committed:
Added .000 to all relevant outputs
Changed expire description to explain ordering
Change update to only allow amount
Added subtract info about earliest expiring ingredient deducted first

mudkip8 committed:
Merge pull request #183 from mudkip8/UserGuide

Some UG changes

mudkip8 committed:
Changed wrong hyperlink label

mudkip8 committed:
Minor PPP update

mudkip8 committed:
Slight change to sequence diagram

mudkip8 committed:
Merge pull request #184 from mudkip8/UserGuide

Changed wrong hyperlink label

ngoivanessa committed:
Merge branch 'master' into v2.0

ngoivanessa committed:
Add ingredient group component add ingredient implementation, udpate IngredientList authorship

ngoivanessa committed:
Merge pull request #185 from ngoivanessa/v2.0

update DG

datn02 committed:
Fix delete command details

datn02 committed:
Fix delete command to the new format

datn02 committed:
Fix UG format of output

datn02 committed:
Fix delete sequence diagram

datn02 committed:
Update PPP

datn02 committed:
Refactor main class

datn02 committed:
Fix minor issues and claim authorship

datn02 committed:
Remove extranous details

datn02 committed:
Merge pull request #186 from datn02/DucAnh-PPPUpdate

datn02 PPP update

mudkip8 committed:
Extra lines to style.puml

mudkip8 committed:
Merge pull request #187 from mudkip8/master

Extra lines to style.puml

Commit:
add details to certain sections of UG

rainish2000 committed:
Merge pull request #188 from nishantrai-nus/edit-userguide

add details to certain sections of UG

Commit:
revert to subtracting by name

Commit:
fix documentation error

rainish2000 committed:
Merge pull request #189 from nishantrai-nus/subtract-by-name

Revert to subtracting by name

Commit:
minor edits to diagrams

rainish2000 committed:
Merge pull request #190 from nishantrai-nus/edit-dg

minor edits to diagrams

AayushMathur7 committed:
Merge branch 'master' of https://github.com/AY2122S1-CS2113T-T09-3/tp

AayushMathur7 committed:
Merge branch 'master' of https://github.com/AY2122S1-CS2113T-T09-3/tp

# Conflicts:
#	docs/DeveloperGuide.md
#	docs/UserGuide.md
#	src/main/java/seedu/situs/ingredients/IngredientList.java

AayushMathur7 committed:
changes to subtract & update; updated UG/DG, PUML Diagrams added

AayushMathur7 committed:
minor updates

AayushMathur7 committed:
Merge pull request #192 from AayushMathur7/master

Added UML, updated DG, minor code changes

datn02 committed:
Fix coding style

datn02 committed:
Merge pull request #193 from datn02/DucAnh-v2.1Fix

Fix coding style

mudkip8 committed:
Fixed checkstyle errors

mudkip8 committed:
Merge branch 'master' of https://github.com/mudkip8/tp

# Conflicts:
#	src/main/java/seedu/situs/ingredients/IngredientList.java

ngoivanessa committed:
Merge branch 'master' into v2.0

Commit:
add ascii art

Commit:
add ASCII art

Commit:
add logger statements

Commit:
add authorship flags

Commit:
minor fixes

Commit:
fix test style violation

Commit:
add ppp for nishant

rainish2000 committed:
Merge pull request #194 from nishantrai-nus/nishant-ppp

Add nishant's ppp and some minor additions to code

Commit:
update aboutus.md

rainish2000 committed:
Merge pull request #195 from nishantrai-nus/nishant-ppp

update aboutus.md

mudkip8 committed:
Changed sequence diagrams to match add colours

mudkip8 committed:
Merge pull request #196 from mudkip8/DeveloperGuide

Developer guide

Commit:
minor addition to find section

rainish2000 committed:
Merge pull request #197 from nishantrai-nus/edit-userguide

minor addition to find section

mudkip8 committed:
Standardized Class colours for seq. diagrams

mudkip8 committed:
Merge pull request #198 from mudkip8/DeveloperGuide

Standardized Class colours for seq. diagrams

mudkip8 committed:
Removed johndoe.md

mudkip8 committed:
Merge pull request #199 from mudkip8/DeveloperGuide

Removed johndoe.md

datn02 committed:
Fix minor detail

datn02 committed:
Merge branch 'master' into DucAnh-v2.1PPPUpdate

datn02 committed:
Add profile picture

datn02 committed:
Merge pull request #200 from datn02/DucAnh-v2.1PPPUpdate

v2.1 PPP and AboutUs update

ngoivanessa committed:
add PPP

ngoivanessa committed:
Merge pull request #201 from ngoivanessa/v2.0

Add PPP

ngoivanessa committed:
Merge branch 'master' into v2.0

AayushMathur7 committed:
minor changes

AayushMathur7 committed:
Merge branch 'master' of https://github.com/AY2122S1-CS2113T-T09-3/tp

AayushMathur7 committed:
updated ppp

AayushMathur7 committed:
Merge pull request #202 from AayushMathur7/master

updates to ppp

Commit:
minor edits

rainish2000 committed:
Merge pull request #203 from nishantrai-nus/edit-userguide

minor edits

datn02 committed:
Add PR information into PPP

datn02 committed:
Merge pull request #204 from datn02/DucAnh-v2.1PPPUpdate2

Add PR information into PPP

datn02 committed:
Fix PPP to be 2 pages long

datn02 committed:
Merge pull request #205 from datn02/DucAnh-v2.1PPPUpdate3

Fix PPP to be 2 pages long

datn02 committed:
Fix output messages format

datn02 committed:
Merge pull request #206 from datn02/DucAnh-v2.1DGFix

Fix output messages format

ngoivanessa committed:
Merge branch 'master' of https://github.com/AY2122S1-CS2113T-T09-3/tp into v2.0

Commit:
edit interaction diagram

Commit:
fix typo

Commit:
update interaction sequence diagram

Commit:
fix bug

Commit:
add find and expire sections with diagrams

rainish2000 committed:
Merge pull request #207 from nishantrai-nus/edit-dg

add find and expire implementations

Commit:
minor fixes

rainish2000 committed:
Merge pull request #208 from nishantrai-nus/edit-dg

minor fixes

Commit:
ppp edits

rainish2000 committed:
Merge pull request #209 from nishantrai-nus/nishant-ppp

ppp edits for nishant

mudkip8 committed:
Changed newest release link

mudkip8 committed:
Merge pull request #210 from mudkip8/master

Changed newest release link in UG

mudkip8 committed:
Fixed a formatting issue with headers

mudkip8 committed:
Merge pull request #211 from mudkip8/master

Fixed a formatting issue with headers

mudkip8 committed:
Added lines to format PDF

mudkip8 committed:
Formatting for PDF

mudkip8 committed:
Merge pull request #212 from mudkip8/master

PDF formatting for UGDG

mudkip8 committed:
More PDF formatting DG

mudkip8 committed:
Merge pull request #213 from mudkip8/master

More PDF formatting DG

mudkip8 committed:
Add newlin to separate add seq diagram and descriptor statement

mudkip8 committed:
Merge pull request #214 from mudkip8/master

Add newline to separate add seq diagram and descriptor statement

mudkip8 committed:
Added 1 page break DG

mudkip8 committed:
Merge pull request #215 from mudkip8/master

Added 1 page break DG

mudkip8 committed:
Updated UG Quickstart output

mudkip8 committed:
Merge pull request #216 from mudkip8/master

Updated UG Quickstart output

datn02 committed:
Claim authorship from expire command

datn02 committed:
Merge pull request #217 from datn02/DucAnh-v2.1Authorship

Claim authorship from expire command

mudkip8 committed:
Fixed minor formatting issue

mudkip8 committed:
Merge pull request #218 from mudkip8/master

Fixed minor formatting issue DG

AayushMathur7 committed:
updated DG

AayushMathur7 committed:
Merge branch 'master' of https://github.com/AY2122S1-CS2113T-T09-3/tp

AayushMathur7 committed:
Merge pull request #219 from AayushMathur7/master

updated DG

ngoivanessa committed:
Merge branch 'master' of https://github.com/AY2122S1-CS2113T-T09-3/tp into v2.0

mudkip8 committed:
Some changes to formatting

mudkip8 committed:
Merge pull request #220 from mudkip8/master

Some changes to formatting to DG

mudkip8 committed:
Trying to fix formatting issues for DG

mudkip8 committed:
Merge pull request #221 from mudkip8/master

Trying to fix formatting issues for DG

mudkip8 committed:
Still trying to fix DG formatting

mudkip8 committed:
Merge pull request #222 from mudkip8/master

Still trying to fix DG formatting

mudkip8 committed:
Third try's the charm for fixing DG formatting

mudkip8 committed:
Merge pull request #223 from mudkip8/master

Third try's the charm for fixing DG formatting

mudkip8 committed:
Attempt 4 at fixing formatting

mudkip8 committed:
Merge pull request #224 from mudkip8/master

Attempt 4 at fixing formatting

mudkip8 committed:
Revert "Attempt 4 at fixing formatting"

This reverts commit b060c1b6a1b3858912a0efe1fced4c12ce074d44.

mudkip8 committed:
Merge pull request #225 from mudkip8/master

Revert "Attempt 4 at fixing formatting"

mudkip8 committed:
Attempt 5 at fixing DG formatting

mudkip8 committed:
Merge pull request #226 from mudkip8/master

Attempt 5 at fixing DG formatting

mudkip8 committed:
Attempt 6 at fixing formatting issue DG

mudkip8 committed:
Merge pull request #227 from mudkip8/master

Attempt 6 at fixing formatting issue DG

mudkip8 committed:
Attempt 7 at fixing DG formatting

mudkip8 committed:
Merge pull request #228 from mudkip8/master

Attempt 7 at fixing DG formatting

mudkip8 committed:
Added some page breaks for DG PDF conversion

mudkip8 committed:
Merge pull request #229 from mudkip8/master

Added some page breaks for DG PDF conversion

ngoivanessa committed:
edited DG and test classes

ngoivanessa committed:
Merge branch 'master' of https://github.com/AY2122S1-CS2113T-T09-3/tp into v2.0

ngoivanessa committed:
Merge pull request #230 from ngoivanessa/v2.0

edit DG and AddCommandTest

mudkip8 committed:
Added page breaks

mudkip8 committed:
Merge branch 'master' of https://github.com/AY2122S1-CS2113T-T09-3/tp

mudkip8 committed:
Merge pull request #231 from mudkip8/master

Added page break

AayushMathur7 committed:
Added J-unit testing

AayushMathur7 committed:
Merge branch 'master' of https://github.com/AY2122S1-CS2113T-T09-3/tp

AayushMathur7 committed:
Merge pull request #232 from AayushMathur7/master

Updated J-Unit testing

datn02 committed:
Fix coding style...

datn02 committed:
Fix update command test not working

datn02 committed:
Merge pull request #233 from datn02/DucAnh-FinalCodingStyleFix

Final coding style fix...
